.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-  "singleQuote": true,
+  "singleQuote": false,
```

Comment:
This change will affect the code quality going forward. We may be distorting Andela-Codacy config

Reply:
i dont think so codacy demands all string to be double-quoted. it a must. most of the boiler plate code from andela failed codacy test